azure-pipelines.yml

```diff
@@ -0,0 +1,209 @@
+    continueOnError: true
+
+  - task: securedevelopmentteam.vss-secure-development-tools.build-task-credscan.CredScan@2
+    displayName: Run CredScan
+    inputs:
+      debugMode: false
+    continueOnError: true
+
+  - task: VSBuild@1
+    displayName: Restore packages
+    inputs:
+      msbuildArgs: '/m /t:restore'
+      platform: $(BuildPlatform)
+      configuration: $(BuildConfiguration)
+
+  - task: VSBuild@1
+    displayName: 'Build solution PortabilityTools.sln'
+    inputs:
+      msbuildArgs: '/m /t:build;pack /bl:$(Build.SourcesDirectory)\bin\$(BuildConfiguration)\msbuild.binlog /p:DeployExtension=False'
+      platform: $(BuildPlatform)
+      configuration: $(BuildConfiguration)
+
+  - task: VSBuild@1
+    displayName: Build apiport global tool
+    inputs:
+      solution: src/ApiPort/ApiPort/ApiPort.csproj
+      msbuildArgs: '-t:pack -p:GlobalTool=true'
+      platform: $(BuildPlatform)
+      configuration: $(BuildConfiguration)
+
+  - script: dotnet publish src/ApiPort/ApiPort/ApiPort.csproj -r win7-x64 -f netcoreapp2.1 --self-contained true
+    displayName: publish self-contained apiport - win7-x64
+
+  - script: dotnet publish src/ApiPort/ApiPort/ApiPort.csproj -r win7-x86 -f netcoreapp2.1 --self-contained true
+    displayName: publish self-contained apiport - win7-x86
+
+  - powershell: |
+      # Extract rule Ids into a plaintext file that is expected by the Roslyn Analyzer build task
+      $xdoc = [xml](Get-Content ".\rules.ruleset")
+      $xdoc.RuleSet.Rules.Rule.Id | Out-File "bin\RoslynAnalyzerSuppressions.txt"
+
+      # Add NoWarn Ids
+      $xdoc = [xml](Get-Content ".\Directory.Build.props")
+      $xdoc.Project.PropertyGroup.NoWarn -split "," | Select -skip 1 | Where {$_} | Add-Content ".\bin\RoslynAnalyzerSuppressions.txt"
+    displayName: Extract suppression ids for Roslyn Analyzer task
+
+  - task: VSTest@2
+    displayName: Test with VSTest
+    inputs:
+      testAssemblyVer2: |
+        **\*test*.dll
+        !**\obj\**
+        !**\PortabilityServiceIntegrationTests.dll
+      runOnlyImpactedTests: false
+      vsTestVersion: '16.0'
+      runInParallel: false
+      runTestsInIsolation: false
+      codeCoverageEnabled: false
+      platform: $(BuildPlatform)
+      configuration: $(BuildConfiguration)
+      diagnosticsEnabled: True
+
+  - task: DotNetCoreCLI@2
+    displayName: Test with dotnet test
+    inputs:
+      command: test
+      projects: |
+        tests/**/*[Tt]ests/*.csproj
+        !tests/ApiPort/ApiPortVS.Tests/ApiPortVS.Tests.csproj
+        !**\PortabilityServiceIntegrationTests.dll
+      arguments: --configuration $(BuildConfiguration) --no-build
+
+  - task: CopyFiles@2
+    displayName: Copy Analysis files to Staging
+    inputs:
+      SourceFolder: bin\$(BuildConfiguration)
+      Contents: |
+        ApiPort\**\*
+        ApiPort.Offline\**\*
+        ApiPort.Vsix\*.vsix
+        raw\**\*
+        *.binlog
+      TargetFolder: $(Build.StagingDirectory)\drop
```

Comment:
No. Having an artifact name that is inconsistent from build to build makes for harder to use artifacts. I changed it from `$(Build.BuildNumber)` to just `drop` so that it was the same across builds.